packages/eslint-config-udemy-website/index.js

```diff
@@ -10,7 +10,6 @@ module.exports = {
     plugins: [
         'udemy',
         'filenames',
-        'lodash',
```

Comment:
I removed the lodash plugin introduced in https://github.com/udemy/js-tooling/pull/10 because the import-blacklist rule that I wrote is a generalized version of it.

BTW, what is the intended way to remove a dependency? There's no "lerna remove", as far as I know. I did `cd packages/eslint-config-udemy/website && yarn remove eslint-plugin-lodash`...

Reply:
Cool, I've documented this in https://github.com/udemy/js-tooling/blob/feature/import-blacklist/README.md.